docs/2024/text-phrases/index.md

```diff
@@ -3,29 +3,51 @@ sidebar_position: 1
 title: Introduction
 slug: /2024/text-phrases/
 ---
+
 <!--
 SPDX-License-Identifier: CC-BY-SA-4.0
 
-SPDX-FileCopyrightText: 2024 Valens Niyonsenga <email.here>
+SPDX-FileCopyrightText: 2024 Valens Niyonsenga <valensniyonsenga2003@gmail.com>
 -->
 
 ## Author
 
-[Valens Niyonsenga](https://github.com/valens200)
+[Valens NIYONSENGA](https://github.com/valens200)
 
 ## Contact info
 
-- [Email](mailto:email.here)
-- [LinkedIn](https://linkedin.com/in/my-user)
+- [Email](valensniyonsenga2003@gmail.com)
```

Comment:
Please change to email link as it causes build failure.

```suggestion
- [Email](mailto:valensniyonsenga2003@gmail.com)
```

Reply:
Sure @GMishx I will take a look into this.